force-app/main/default/classes/ServiceSessionSelector.cls

```diff
@@ -15,6 +15,26 @@ public with sharing class ServiceSessionSelector {
     @TestVisible
     private FieldBucketSelector bucketSelector = new FieldBucketSelector();
 
+    public List<ServiceSession__c> getFutureServiceSessionsForServiceSchedule(
+        Id serviceScheduleId
+    ) {
+        List<ServiceSession__c> serviceSessions = new List<ServiceSession__c>();
+
+        if (!Schema.SObjectType.ServiceSession__c.isAccessible()) {
```

Comment:
I think I'm inclined to skip this one, both for pragmatically getting the WI closed and also because this matches the other methods in this class currently